src/main/java/io/kuzzle/sdk/security/KuzzleSecurity.java

```diff
@@ -361,7 +361,7 @@ public void updateRole(@NonNull final String id, final JSONObject content, final
         @Override
         public void onSuccess(JSONObject response) {
           try {
-            listener.onSuccess(response.getJSONObject("result").getString("_id"));
+            listener.onSuccess(new KuzzleRole(KuzzleSecurity.this.kuzzle, response.getJSONObject("result").getString("_id"), response.getJSONObject("result")));
```

Comment:
http://kuzzle.io/api-reference/?websocket#updaterole doesn't say there's a `_source` object in the result